rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -26,7 +26,18 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
     def packerCommand = [baseCommand, "packer", "build", "-color=false"]
     parameterMap.each { key, value ->
       if (key && value) {
-        def keyValuePair = value instanceof String && value.contains(" ") ? "$key=\"$value\"" : "$key=$value"
+        def cleansedValue = value instanceof String ? value.trim() : value
```

Comment:
This change originally concerned me, but I think it's ok. I ran a test against gce using a space-separated list of package names and it was handled properly. I _think_ this may have been some historical logic left over from when we were doing builds using rush & containers (I also recall that we were tokenizing the command on spaces, I _think_ just before sending it to rush, but that was a while back and the memories are fading).

Also, why do we need to handle the case where the param has leading/trailing quotes? Is there some path that creates that situation (other than a user actually adding those quotes into the packages field of the bake stage ui; I'd think quoting values in just about any of the fields throughout the ui would create problems)? If there isn't a path that leads to having those surrounding quotes, let's just drop the decorating & cleansing logic altogether.

@gardleopard Any concerns with this change?